app/controllers/application_controller.rb

```diff
@@ -9,4 +9,8 @@ def current_tenant
   def authorize_admin!
     redirect_to(root_path) unless true_user&.role == User::ADMIN_ROLE
   end
+
+  def authorize_super_admin!
+    redirect_to(root_path) unless true_user&.role == User::SUPER_ADMIN_ROLE
```

Comment:
this check should be inside ```User``` model. 
here code should look like:
```ruby
true_user&.super_admin?
```

Reply:
Done.
Can I get your approval now?)